src/libxrpl/protocol/LedgerFormats.cpp

```diff
@@ -365,6 +365,21 @@ LedgerFormats::LedgerFormats()
         },
         commonFields);
 
+    add(jss::Credential,
+        ltCREDENTIAL,
+        {
+            {sfSubject,              soeREQUIRED},
+            {sfIssuer,               soeREQUIRED},
+            {sfCredentialType,       soeREQUIRED},
+            {sfSignature,            soeOPTIONAL},
```

Comment:
`Signature` field is not included in the specs in the `Credential` object.

Reply:
fixed. Also `sfIssuer `field in this transaction changed to `sfPublicKey `field (which should be public key of the issuer for checking `Signature`)